.travis.yml

```diff
@@ -29,7 +29,6 @@ before_install:
   - |
     if [ "$TRAVIS_OS_NAME" == freebsd ]; then
       sudo pkg install -y evdev-proto libgudev libxml++
-      sudo pkg install -y automake libtool pkgconf gmake
```

Comment:
Can we fixup the first commit with this change instead of introducing it separately?

Reply:
For easier re-review. I've squashed all FreeBSD commits.